### PR TITLE
Create 20-dancing-with-htmx.pod

### PR DESCRIPTION
**Pull request by davorg**

### CODE REVIEW

File: danceradvent/public/articles/2023/20-dancing-with-htmx.pod [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+      return 404 unless $id;
+      return 404 if $id =~ /\D/;
+
+      my $sch = HTMXmas::Schema->connect(...);
+      my $rs  = $sch->resultset('Child');
+
+      $rs->find($id)->delete;
+
+      $row->delete;
```
Comment (cromedome) [inline danceradvent/public/articles/2023/20-dancing-with-htmx.pod:191]: Did you originally have one line to find and another to delete? I think you can safely delete this.

File: danceradvent/public/articles/2023/20-dancing-with-htmx.pod [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+Which brings us neatly back to the Dancer app. We need to add two new routes
+to handle these icon - C<post '/update/:id/'> will save the data to the
+database (and remove the form from the table) and C<get '/reset/:id'> will
+undo any changes and remove the form. They look like this:
+
+    post '/update/:id' => sub {
+      my $id = route_parameters->get('id');
+
+      my $object = from_json($request->body);
```
Comment (cromedome) [inline danceradvent/public/articles/2023/20-dancing-with-htmx.pod:292]: Drop the leading $ from request here since it's a keyword.

**Review by cromedome — CHANGES_REQUESTED** [review]
I love this! It's fun and easy to follow along, and makes me want to drop everything else to go play with htmx now :)

File: danceradvent/public/articles/2023/20-dancing-with-htmx.pod (hunk 2 above)
Comment (davorg) [reply danceradvent/public/articles/2023/20-dancing-with-htmx.pod:292]: Ah yes. That was simplified from a longer version where that line of code was in a subroutine that took the request as a parameter. Thanks.

File: danceradvent/public/articles/2023/20-dancing-with-htmx.pod (hunk 1 above)
Comment (davorg) [reply danceradvent/public/articles/2023/20-dancing-with-htmx.pod]: I went back and forth on those lines so many times. And ended up in an inconsistent state :-0

**Comment by cromedome** [comment]
Merged, thanks!

**Pull request closed without merging**